src/utils/saml.ts

```diff
@@ -0,0 +1,21 @@
+import { DOMParser } from "xmldom";
+
+export interface ISamlResponse {
+  RelayState: string;
```

Comment:
this field is not used in the code below, you can avoid declaring it in the interface

Reply:
fixed in 1145b45